tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/tuples/TupleAccessExprNegativeTest.java

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+package org.ballerinalang.test.types.tuples;
+
+import org.ballerinalang.test.BCompileUtil;
+import org.ballerinalang.test.CompileResult;
+import org.testng.annotations.Test;
+
+import static org.ballerinalang.test.BAssertUtil.validateError;
+import static org.testng.Assert.assertEquals;
+
+/**
+ * Tuple access with indexes negative test.
```

Comment:
Let's add the version too.

Reply:
Meant to say the `@since` annotation.

Reply:
I will add

Reply:
Removed this file according to this https://github.com/ballerina-platform/ballerina-lang/pull/37008#discussion_r931772355